src/demos/approval-magic/README.md

```diff
@@ -137,15 +137,15 @@ The `Deployed to` address should be assigned to `EXCH_ADDR`.
 
 #### Fund the Exchanged Contract
```

Comment:
Sorry, I seem to have misguided you. I noticed a typo in this heading, not in everything else.